modules/stnBidAdapter.md

```diff
@@ -0,0 +1,77 @@
+#Overview
+
+Module Name: STN Bidder Adapter
+
+Module Type: Bidder Adapter
+
+Maintainer: hb@stnvideo.com
+
+
+# Description
+
+Module that connects to STN's demand sources.
+
+The STN adapter requires setup and approval from the STN. Please reach out to hb@stnvideo.com to create an STN account.
+
+The adapter supports Video(instream) & Banner.
+
+# Bid Parameters
+## Video
+
+| Name | Scope | Type | Description                                                       | Example
+| ---- | ----- | ---- |-------------------------------------------------------------------| -------
+| `org` | required | String | STN publisher Id provided by your STN representative              | "0987654321abcdef87654321"
+| `floorPrice` | optional | Number | Minimum price in USD. Misuse of this parameter can impact revenue | 2.00
+| `placementId` | optional | String | A unique placement identifier                                     | "12345678"
+| `testMode` | optional | Boolean | This activates the test mode                                      | false
+| `currency` | optional | String | 3 letters currency                                                | "EUR"
+
+# Test Parameters
+```javascript
+var adUnits = [{
+        code: 'dfp-video-div',
+        sizes: [
+            [640, 480]
+        ],
+        mediaTypes: {
+            video: {
+                playerSize: [
+                    [640, 480]
+                ],
+                context: 'instream'
+            }
+        },
+        bids: [{
+            bidder: 'stn',
+            params: {
+                org: '0987654321abcdef87654321', // Required
+                floorPrice: 2.00, // Optional
+                placementId: 'video-test', // Optional
+                testMode: false // Optional
```

Comment:
Please update your endpoint to return valid bids with these params

Reply:
Setting the `testMode` param to `true` inside the bidder params enables the test ads

Reply:
I am seeing this response: `{"params":{"sessionId":"141c6e86e34c45"}}`. 

Could you look into what is necessary to return a valid bid response? As well, let's update your markdown file to set `testMode` to `true` with a comment so others know how to test

Reply:
Hi @ncolletti, I've updated the test mode params in the documentation. Can you, please, verify if it works on your end now?